interfaces/acados_matlab_octave/ocp_set_ext_fun.m

```diff
@@ -43,11 +43,11 @@
 
 % set paths
 acados_mex_folder = fullfile(acados_folder, 'interfaces', 'acados_matlab_octave');
-acados_include = ['-I' acados_folder];
+acados_include = ['-I' fullfile(acados_folder,'include')];
```

Comment:
Good catch, the folders here should be correct!
Probably, it should be adjusted in `interfaces/acados_matlab_octave/sim_set_ext_fun.m` as well

Reply:
I updated the above mentioned filed, but I have not tested this my self. Hopefully this is covered by the CI-test